packages/go_router_builder/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.8.2
+
+- Fixes an issue when enum params is required
```

Comment:
I assume the reason we can't slap on a `!` is due to they may not be a required paremeter

```suggestion
- Fixes an issue when enum params are not required
```